app/views/journalUpload.html

```diff
@@ -84,7 +84,6 @@ <h2 class="page-title" translate>
           <option value="full" translate>Full/Overpaid</option>
           <option value="partial" translate>Partial</option>
           <option value="full-partial" translate>Full/Partial</option>
-          <option value="none" translate>Not Paid</option>
```

Comment:
I was asked to remove this as it would be confusing to leave in. Anything they should see on the Journal Upload page should technically be "Paid".